src/Components/Lobby/Lobby.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import Lobby from './Lobby';
+
+test('renders learn react link', () => {
+  render(<Lobby />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
+
+// It should render a welcome greeting
```

Comment:
LOVE the pseudo code start for the testing file. Even though we aren't writing the actual tests just yet, it's great that we're actively keeping testing in the front of our minds as we go along.

Reply:
I agree, it's important to keep testing in mind while we are writing.